localstack/testing/pytest/fixtures.py

```diff
@@ -373,7 +375,9 @@ def factory(**kwargs):
 
         tables.append(kwargs["table_name"])
 
-        return create_dynamodb_table(**kwargs)
+        fn = create_dynamodb_table(**kwargs)
+        dynamodb_wait_for_table_active(kwargs["table_name"])
```

Comment:
As I understand the `create_dynamodb_table` already has a [wait included](https://github.com/localstack/localstack/blob/master/localstack/utils/aws/aws_stack.py#L989-L993) and `wait_for_active` is set by default. 
I think we could remove it here?

Reply:
Good catch 👍